source/includes/databases-collections/databases-collections.php

```diff
@@ -63,13 +72,15 @@
 
 // end-collection-settings
 
-// Instructs the library to prefer New York data center reads using a tag set
+// Instructs the library to prefer reads from secondary replica set members
+// located in New York but fall back to those in San Francisco
```

Comment:
```suggestion
// located in New York, followed by a secondary in San Francisco, and
// lastly fall back to any secondary.
```

This captures the effect of the empty tag set.

Please adjust the RST paragraph above this code example accordingly. If you want to borrow some language from the existing server manual, there's a nicely worded tip in [Order of Tag Matching](https://www.mongodb.com/docs/v5.2/core/read-preference-tags/#order-of-tag-matching).